src/main.rs

```diff
@@ -148,6 +148,8 @@ fn run_cli<W: Write, R: Read, L: OutputPin, SPI, PADS, const I: usize>(
             #[command(subcommand)]
             command: Option<BootargsCommand<'a>>,
         },
+        ///print the infomation in configs.bootargs
+        Print, 
```

Comment:
```suggestion
/// Print the infomation in configs.bootargs.
        Print, 
```
All our code should follow Rust comment code style conventions, that means, there should be one space after document comment `///`, and it should be an English sentence with a beginning of capital letters and a period `.` on the end.

Reply:
Thanks for your pointer! I will be more careful about the Rust comment code style conventions.